pkg/api/server/register_secrets.go

```diff
@@ -89,6 +89,25 @@ func (s *APIServer) registerSecretHandlers(r *mux.Router) error {
 	//   '500':
 	//     "$ref": "#/responses/internalError"
 	r.Handle(VersionedPath("/libpod/secrets/{name}/json"), s.APIHandler(compat.InspectSecret)).Methods(http.MethodGet)
+	// swagger:operation GET /libpod/secrets/{name}/exists libpod SecretExistsLibpod
+	// ---
+	// tags:
+	//  - secrets
+	// summary: Secret exists
+	// parameters:
+	//  - in: path
+	//    name: name
+	//    type: string
+	//    required: true
+	//    description: the name or ID of the secret
+	// produces:
+	// - application/json
+	// responses:
+	//   '200':
```

Comment:
I think that a `204` for the positive and a `404` for the negative case would be more RESTful.  That's what the other `/exists` endpoints do as well AFAICS.

Reply:
Not a samll nit :)
Fixed and adjusted both the server and client sides